app/src/main/res/layout/activity_offer.xml

```diff
@@ -19,6 +19,22 @@
         tools:listitem="@layout/offer_header"
         tools:paddingTop="?attr/actionBarSize" />
 
+    <Button
+        android:id="@+id/signButton"
+        style="?attr/materialButtonContainedLargeStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom"
+        android:layout_marginHorizontal="@dimen/base_margin_double"
+        android:layout_marginTop="@dimen/base_margin_quadruple"
+        android:text="@string/OFFER_SIGN_BUTTON"
+        android:visibility="gone"
+        app:icon="@drawable/ic_bank_id"
+        app:iconGravity="textEnd"
+        app:iconPadding="@dimen/base_margin_double"
+        app:layout_anchorGravity="bottom"
```

Comment:
How does `anchorGravity` work? 🤔 I'm not familiar with it at all, and the documentation did not help me understand it better 🙈

Reply:
Sorry will remove this, was trying some stuff out and this slipped through. Good catch.